modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc

```diff
@@ -50,11 +50,10 @@ For more information, see link:https://www.apollographql.com/docs/react/essentia
 === Global Update Functions
 
 Apollo client holds all mutation parameters in memory.
-Offline Client will save and restore all mutations with exact parameters
-when application is restarted. Update functions supplied to mutations cannot be saved in the cache. 
-As result all optimisticResponses will disappear from application after restart.
+An offline Apollo client will continue to store mutation parameters and once restarted, it will restore all mutations to memory. Any Update Functions that are supplied to mutations cannot be cached by an Apollo client resulting in the loss of all optimisticResponses after a restart. _Update functions_ supplied to mutations cannot be saved in the cache. 
+As result all _optimisticResponses_ will disappear from application after restart and it will appear back only when application becomes online and they will be replicated to server.
```

Comment:
again I'm confused between 'starting' and 'online', maybe i'm missing something?

Reply:
The application can be restarted and still be like offline or something. 

Online/Offline will be independent from:
- client lifecycle
- Application restarts.